snake.py

```diff
@@ -0,0 +1,58 @@
+from turtle import Turtle
+import json
+data_file = open('constants.json')
+CONSTANTS = json.load(data_file)
+
+
+class Snake:
+
+    def __init__(self):
+        self.segments = []
+        self.create_snake()
+        self.head = self.segments[0]
+
+    # create_snake  calls add_segment() at to create snake
+    def create_snake(self):
+        for position in CONSTANTS["STARTING_POSITIONS"]:
+            self.add_segment(position)
+
+    # add_segment() creates Turtle object called snake_fragment and appends to fragment list
+    def add_segment(self, position: tuple):
+        snake_fragment = Turtle(shape="square")
+        snake_fragment.penup()
+        snake_fragment.color("white")
```

Comment:
And here. But I think you get it.

Reply:
flushed out the rest of the constants, and  I included a json file to read and write the high score to